docker-compose.yaml

```diff
@@ -14,9 +14,9 @@ services:
     entrypoint:
       - tenzir-node
     volumes:
-      - tenzir-node:/var/lib/tenzir/
-      - tenzir-node:/var/log/tenzir/
-      - tenzir-node:/var/cache/tenzir/
+      - tenzir-lib:/var/lib/tenzir/
+      - tenzir-log:/var/log/tenzir/
+      - tenzir-cache:/var/cache/tenzir/
```

Comment:
Quick question: Does it make sense to just remove this for the cache? We wipe it on startup anyways, so there's hardly any need to persist it.

Reply:
Makes sense to me: This would cause the cache to be bound to the given container. So a `docker (compose) restart` would keep it, but as soon as a new container is created, it's gone.

Reply:
However the cache is shared between and `tenzir-node` and `tenzir` (the CLI container).
Is that necessary?

Reply:
Hasn't been since v4.21 or so.